### ISSUE TITLE
`docker.io/envoyproxy/envoy:v1.23.0` `arm64` image fails with `./docker-entrypoint.sh: 29: exec: su-exec: Exec format error`

### ISSUE THREAD

**Issue by mbana** [envoyproxy/envoy#22384]
**Host:**

```sh
$ uname -a
Linux ubuntu-free-x86 5.15.0-1013-oracle #17-Ubuntu SMP Fri Jun 24 10:14:16 UTC 2022 aarch64 aarch64 aarch64 GNU/Linux
$ docker run --rm -it docker.io/envoyproxy/envoy:v1.23.0            
/docker-entrypoint.sh: 29: exec: su-exec: Exec format error
```

**Debug Log:**

```sh
$ docker run --rm -it --entrypoint=/bin/bash docker.io/envoyproxy/envoy:v1.23.0
$ file /usr/local/bin/envoy 
/usr/local/bin/envoy: ELF 64-bit LSB shared object, x86-64, version 1 (SYSV), dynamically linked, interpreter /lib64/ld-linux-x86-64.so.2, for GNU/Linux 3.2.0, BuildID[sha1]=ce49c7f65668a22b80d1e83c35d170741bb8d46a, stripped
$ which envoy
/usr/local/bin/envoy
$ ./docker-entrypoint.sh 
./docker-entrypoint.sh: 29: exec: su-exec: Exec format error
$ uname -a
Linux 47906637c567 5.15.0-1013-oracle #17-Ubuntu SMP Fri Jun 24 10:14:16 UTC 2022 aarch64 aarch64 aarch64 GNU/Linux
```

From the above it's obvious it's shipping an x86_64 image:

```sh
$ file /usr/local/bin/envoy 
/usr/local/bin/envoy: ELF 64-bit LSB shared object, x86-64, version 1 (SYSV), dynamically linked, interpreter /lib64/ld-linux-x86-64.so.2, for GNU/Linux 3.2.0, BuildID[sha1]=ce49c7f65668a22b80d1e83c35d170741bb8d46a, stripped
```

**Comment by phlax** [comment]
hi @mbana thanks for reporting - im going to close this as duplicate of #22262 which was inadvertently closed

i expect to have a new release out by early next week

**Comment by mbana** [comment]
Thank you for the quick response.

Can you not make an interim build for `docker.io/envoyproxy/envoy:v1.23.0` tagged `envoy:v1.23.1` or `latest` or so. One week a long wait ...

**Comment by phlax** [comment]
not as it stands - currently publishing Docker images requires a new patch version, and there are a bunch of other things that need to go into that

there has been some discussion about separating the container publishing from release publishing, but that is not the case today

**Comment by mbana** [comment]
@phlax and @GJKrupa, why was this closed?

Could you please try running an arm64 image, ideally on an arm64 host as well, and confirm that it is working.

Thanks a lot.

**Comment by phlax** [comment]
closed as duplicate

this will be fixed once #22813 lands
